### ISSUE TITLE
Missing sourcemaps in distribution?

### ISSUE THREAD

**Issue by mjpizz** [0no-co/wonka#119]
I'm using `urql` (3.x) with `create-react-app` (5.x) and it appears the dev server is unable to find proper sourcemaps for wonka in particular:

```
WARNING in ./node_modules/wonka/dist/wonka.mjs
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '[path_to_project]/node_modules/src/callbag.ts' file: Error: ENOENT: no such file or directory, open '[path_to_project]/node_modules/src/callbag.ts'
```

This means debugging urql/wonka becomes more difficult, and it also makes a lot of noise for the React devserver logs that can make other issues easier to miss.

Any ideas what might be going on? Does wonka intentionally not ship with sourcemaps currently?

**Comment by tanmen** [comment]
I too am struggling with this issue.
I have created a PR and I hope it gets resolved!
#120

**Comment by kitten** [comment]
Resolved by #120. I'll take care of a new minor release later today
